sport/app/rugby/feed/OptaFeed.scala

```diff
@@ -4,22 +4,40 @@ import common.{ExecutionContexts, Logging}
 import play.api.Play.current
 import play.api.libs.ws.{WSRequest, WS}
 import rugby.jobs.RugbyStatsJob
-import rugby.model.{ScoreEvent, Match}
+import rugby.model.{ScoreEvent, Match, MatchStat}
 
 import scala.concurrent.Future
 import scala.xml.XML
 
+sealed trait OptaEvent {
+  def competition: String
+  def season: String
+}
+
+case object WarmupWorldCup2015 extends OptaEvent {
+  override val competition = "3"
+  override val season = "2016"
+}
```

Comment:
We can't use this but let's chat tomorrow with people and give an idea of what it can do. If they like it we might be able to get sign off etc.